showpass/Showpass-E2E-tests
# Fix the Interac checkout (payment) method

**Description:**
The 'Click' action was accidentally removed from the script before, which caused a new failre.
Added it back.

**How to test:**
Run the following spec and ensure all tests pass: cypress/e2e/events/purchases.cy.js

**Test execution results:**

[Cypress Cloud results](https://cloud.cypress.io/projects/e9axi7/runs/196/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D)
